src/content/en/updates/2018/06/webar-chacmool.md

```diff
@@ -0,0 +1,121 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Chacmool, an educational web demo, shows how easily web based AR can build an engaging experience.
+
+{# wf_updated_on: 2018-06-14 #}
+{# wf_published_on: 2018-06-14 #}
+{# wf_tags: immersive-web,webar,webxr #}
+{# wf_featured_image: /web/updates/images/2018/06/webar-chacmool.jpg #}
+{# wf_featured_snippet: Chacmool, an educational web demo, shows how easily web based AR can build an engaging experience. #}
+{# wf_blink_components: Blink>WebVR #}
```

Comment:
Should this be WebAR?

Reply:
No.

Reply:
Hmm, OK, but it was added in `commonTags.json` below.

Reply:
A subject index is different from a list of blink components.

Reply:
The point is that an index is free to use whatever terms are in a readers's head and is not restricted to what are used in the product "paperwork".